docs/multicluster/architecture.md

```diff
@@ -80,18 +97,53 @@ Service `cluster1-ns-foo-service`, `cluster2-ns-foo-service` and associated Endp
 ServiceImport `ns/foo` locally if they don't exist or updates them if the resources have already
 been created by the Importer earlier.
 
-## Antrea Multi-cluster Service
-
-Antrea Multi-cluster Controller only supports Service of type ClusterIP at this moment. In
-order to support multi-cluster Service access between member clusters, Antrea requires member
-clusters' Pod IPs are reachable and no overlapping between all member clusters.
-
-When Antrea Multi-cluster Controller in member cluster watches ResourceImport's creation event
-in leader cluster, it will create multi-cluster Service, Endpoints and ServiceImport locally.
-The Service Ports definition will be the same as exported Services, the Endpoints will be Pod
-IPs from all member clusters. The new created Antrea Multi-cluster Service is just like a regular
-Kubernetes Service, so Pods in a member cluster can access the multi-cluster Service as usual without
-any extra setting.
+## Antrea Multi-cluster Gateway
+
+Antrea Multi-cluster supports Service connectivity directly since 1.7.0. User can choose one of K8s Nodes
```

Comment:
Antrea started to support Multi-cluster Gateway since v1.7.0.

Reply:
Done